toxicity/test/test_linear_predictor.py

```diff
@@ -0,0 +1,52 @@
+import numbers
+import unittest
+import pathmagic  # noqa
```

Comment:
Is #noqa needed here? If yes what is the purpose of pathmagic?

Reply:
yes, it is needed in order to be compliant with the PEP8.